resources/table/styles/table.module.css

```diff
@@ -0,0 +1,30 @@
+.table {
```

Comment:
Don't put the styles in the folder `styles` let then be at the same level of the component.